Bindings/Python/CMakeLists.txt

```diff
@@ -29,25 +40,17 @@ add_custom_command(
         -I${OpenSim_SOURCE_DIR}/Bindings/
         -I${Simbody_INCLUDE_DIR}
         -o ${swig_output_cxx_file_fullname}
-        -outdir "${PYTHON_PACKAGE_BINARY_DIR}/opensim"
+        -outdir "${CMAKE_CURRENT_BINARY_DIR}"
         ${swig_interface_file_fullname}
     DEPENDS ${swig_interface_file_fullname}
         "${OpenSim_SOURCE_DIR}/Bindings/opensim.i"
         "${OpenSim_SOURCE_DIR}/Bindings/OpenSimHeaders.h"
```

Comment:
@aymanhab do you know which *.i file he edited? @aseth1. Both *.i files are listed as dependencies of this command, so should invoke swig to rerun. I am near certain that @aseth1 was using an intermediate version of the code that did not have the above two lines.